files/es/web/api/notification/requestpermission/index.md

```diff
@@ -0,0 +1,98 @@
+Notification.requestPermission(callback);
+```
+
+### Parámetros
+
+- `callback` {{optional_inline}} {{deprecated_inline}}
+  - : Una callback opcional que es ejecutado con el valor del permiso. Obsoleto y reemplazado por el valor de retorno de la promesa.
+
+### Retorno
+
+Una {{jsxref("Promise")}} que se convierte en una {{domxref("DOMString")}} con el permiso escogido por el usuario. Los valores posibles para este son 
+
+- `granted`
+- `denied`
+- `default`
+
+## Ejemplos
+
+Asume este HTML básico:
+
+```html
+<button onclick="notifyMe()">Notify me!</button>
+```
+
+Es posible enviar una notificacion de la siguiente forma - aqui presentamos un codigo bastante verboso y completo que podrias usar si quisieras primero comprobar si las notificaciones son soportadas, luego comprobar si el permiso ha sido concedido para el origen actual para enviar notificaciones y luego solicitar permiso de ser requerido, antes de enviar una notificación.
+
+```js
+function notifyMe() {
+  // Comprobamos si el navegador soporta las notificaciones
+  if (!("Notification" in window)) {
+    alert("Este navegador no es compatible con las notificaciones de escritorio");
+  }
+
+  // Comprobamos si los permisos han sido concedidos anteriormente
+  else if (Notification.permission === "granted") {
+    // Si es correcto, lanzamos una notificación
+    var notification = new Notification("Hola!");
+  }
+
+  // Si no, pedimos permiso para la notificación
+  else if (Notification.permission !== "denied") {
+    Notification.requestPermission().then(function (permission) {
+      // Si el usuario nos lo concede, creamos la notificación
+      if (permission === "granted") {
+        var notification = new Notification("Hola!");
+      }
+    });
+  }
+
+  // Por último, si el usuario ha denegado el permiso, y quieres ser respetuoso, no hay necesidad de molestarlo.
+}
+```
+
+Ya no mostramos ejemplos corriendo en tiempo real en esta página, ya que Chrome y Firefox ya no conceden permisos de notificación para ser solicitados desde el origen {{htmlelement("iframe")}}, para seguir con otros navegadores. Para ver un ejemplo en acción, revisa nuestro [Ejemplo de lista de Todos](https://github.com/mdn/to-do-notifications/tree/gh-pages) (véase también [la app corriendo en vivo](https://mdn.github.io/to-do-notifications/).)
+
+> **Nota:** En el ejemplo de arriba generamos notificaciones en respuesta a una acción del usuario (hacer click en un botón). Esta no es la única mejor práctica — no deberias hacer spam a los usuarios con notificaciones que ellos no aceptaron — pero pronto los navegadores explicitamente deshabilitaran las notificaciones que no son ejecutadas en respuesta a una acción del usuario. Firefox ya está haciendo esto desde la version 72, por ejemplo.
+
+## Especificaciones
+
+{{Specifications}}
+
+## Compatibilidad con Navegadores
+
+{{Compat}}
+
+## Véase también
+
+- [Usando la API de Notificaciones](/es/docs/Web/API/Notifications_API/Using_the_Notifications_API)
```

Comment:
```suggestion
- [Usando la API de Notificaciones](/es/docs/Web/API/Notifications_API/Using_the_Notifications_API)

```

Reply:
Just wanted to quickly point out that for some reason, we can't see newlines at the end of files in a PR even if it's actually there. Not quite sure why but this is something I recently noticed, and I trust that **Vallejoanderson** added one, but it's just not showing up

Reply:
I didn't realize that you have told me though